v2/apiserver/internal/core/workers_test.go

```diff
@@ -205,6 +205,82 @@ func TestWorkersServiceUpdateStatus(t *testing.T) {
 	}
 }
 
+func TestWorkersServiceCleanup(t *testing.T) {
```

Comment:
Should we add a success case here? (or intended to be covered by the corresponding sdk test?)

Reply:
Oversight. Nice catch. Fixing.

Reply:
Fixed.